src/components/TheTopNavbar.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar-title>
+      <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
+        <TheUserMenu />
+        <img
+          id="logo"
+          alt="Cuttle logo"
+          src="/img/cuttle_logo_text_white.svg"
+          width="60"
+          height="60"
+          class="ma-md-auto"
+        />
+      </div>
+    </v-toolbar-title>
+    <v-toolbar-items v-if="!isSmallDevice" class="hidden-xs-only">
+      <v-tabs class="pa-2">
+        <v-tab
+          variant="text"
+          v-for="({ text, icon, page }, i) in pageLinks"
+          :data-cy="text"
+          :key="i"
+          :title="text"
+          :to="page"
+        >
+          <v-icon :icon="`mdi-${icon}`" />
+          {{ text }}
+        </v-tab>
+      </v-tabs>
+    </v-toolbar-items>
+  </v-toolbar>
+
+</template>
+
+<script>
+import { getPageLinks } from '../composables/navLink.js';
+import TheUserMenu from '../components/TheUserMenu.vue';
+
+export default {
+name: 'TheTopNavbar',
+  components: {
+    TheUserMenu
+  },
+  props: {
+    isSmallDevice: {
```

Comment:
It might be better to provide this from app and then inject here where you need it if you want the shortened name, alternatively it might be ok just to use the vuetify namespace since they add $vuetify to every component.

Reply:
After taking a closer look at the Vue style guide re: [singleton instance components](https://vuejs.org/style-guide/rules-strongly-recommended.html#single-instance-component-names), these components shouldn't have props, so as @seriouslysean let's remove the `isSmallDevice` prop and instead compute that directly from within this component.

In order to keep the names more brief, as @seriouslysean has suggested, let's rename this `TheHeader` and rename `TheBottomNav` to `TheFooter`.

We are switching to `<script setup>` syntax, so going forward we are going to ask that all new components (including these two) be implemented in the `<script setup>` style.

So:
1. Let's rename this file to `TheHeader.vue`
2. Let's convert it to `<script setup>`
3. Let's remove the `isSmallDevice` prop and compute it inside this component with Vuetify's [useDisplay() composable](https://vuejs.org/style-guide/rules-strongly-recommended.html#single-instance-component-names)